.eslintrc.js

```diff
@@ -82,6 +91,9 @@ module.exports = {
         '@typescript-eslint/no-var-requires': 'off',
         '@typescript-eslint/require-await': 'off',
         '@typescript-eslint/restrict-template-expressions': 'off',
+        'react/jsx-props-no-spreading': 'off',
+        'max-classes-per-file': 'off',
+        'default-param-last': 'off',
```

Comment:
`'react/no-unstable-nested-components'`, `'max-classes-per-file'`, and `'default-param-last'` I chose to disable because the amount of refactoring they would require seemed out of scope for this ticket.

`react/jsx-props-no-spreading` I turned off because I don't think prop spreading is evil.